libqtile/backend/wayland/xwindow.py

```diff
@@ -351,13 +355,22 @@ def __init__(
         qtile: Qtile,
         win: XWindow,
         idle_inhibitor_count: int,
+        x: int | None,
+        y: int | None,
+        width: int | None,
+        height: int | None,
     ):
         surface = win.surface
         Static.__init__(
             self, core, qtile, surface, win.wid, idle_inhibitor_count=idle_inhibitor_count
         )
         self._wm_class = surface.wm_class
 
+        self.conf_x = x
+        self.conf_y = y
+        self.conf_width = width
+        self.conf_height = height
```

Comment:
If these are only used by `self`, could we make them private attributes?

Reply:
We could, but they're public in x11.window.Static.  It's up to you.

Reply:
Yep let's go private. Really the x11 ones ought to be too.

Reply:
Done!